Documentation/ApiOverview/Database/DatabaseUpgrade/Index.rst

```diff
@@ -54,10 +53,12 @@ very likely break your installation.
 More information about the process of upgrading TYPO3 CMS can be found in
 the :ref:`Installation and Upgrade Guide <t3install:upgrade>`.
 
-
+.. index::
+   File; EXT:{extkey}/ext_tables.sql
+   Database; CREATE TABLE statement
```

Comment:
Just write "Database; CREATE TABLE"? Above we have just "INSERT" and "UPDATE", not the additional "statement"